### ISSUE TITLE
Accessibility: Dyslexic Users: Italic font

### ISSUE THREAD

**Issue by shawnthompson** [a11yproject/a11yproject.com#434]
I'd like to share[ a discussion](https://github.com/wet-boew/wet-boew/issues/7631) we are having north of the border about using _italics_ and the effect it has on people with Dyslexia.

In this article, [6 Surprising Bad Practices That Hurt Dyslexic Users](http://uxmovement.com/content/6-surprising-bad-practices-that-hurt-dyslexic-users/), it states:

> Italics are sometimes used to highlight text. But you shouldn’t use italicized text because they make letters hard to read. The letters have a jagged line compared to non-italic fonts. The letters also lean over making it hard for dyslexic users to make out the words [6].

This study, [Good Fonts for Dyslexia](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf), talks about the different fonts used and the effects it has on the participants of the study.

Curious if anyone here has heard or read anything on _italics_ should be avoided for accessibility reasons?

Thanks again, keep up the good work!

**Comment by nattarnoff** [comment]
I'm in agreement (for the most part) with this information. However...

1) They mean pseudo-italics. If you are using a solid font family with an
italic style it should not have the jagged lines.
2) Italics are hard to read for most people, but....the letters that often
give dyslexics trouble (agjJlLqbpd69) in many italic fonts these take on
different shapes that are harder to confuse.

Sooo....
My recommendations:
- Don't use pseudo-italics, have an actual font.
- Don't use at small sizes because all people will struggle to read.
- Italics MIGHT be ok at large font sizes (20px+).

**Comment by shawnthompson** [comment]
Looks like we are removing all italics from our site. Not using `em` and changing the `font-style` for elements like `cite` and `dfn`. There's a lot more on that [thread ](https://github.com/wet-boew/wet-boew/issues/7631)if you want to read more about it.

**Comment by davatron5000** [comment]
Are there any particular resolutions we want to make for this site? I could see a "Quick Tip: Using italics" with @nattarnoff's summation and some of the points from that thread. Outlawing italics is maybe too strong a message, but a "best practices for applying emphasis" would be helpful.

But If no one is interested in writing it, I'll close this issue.

**Comment by timwright12** [comment]
Happy to contribute/repurpose any content from my article about this: http://csskarma.com/blog/dyslexia-typography/

**Comment by svinkle** [comment]
@timwright12 I just read your article. Very interesting topic! It would be great if you were able to contribute some of it to the site here. 👍

**Comment by timwright12** [comment]
@svinkle I'd be happy to contribute! I'll try and carve out some time.

**Comment by svinkle** [comment]
@timwright12, any update on the status of this article?

**Comment by timwright12** [comment]
@svinkle nothing quite yet. Just made a big move, hoping to carve out time soon.

**Comment by svinkle** [comment]
@timwright12 Cool, no worries! Just wanted to check in. 👍

**Comment by Martin-Pitt** [comment]
I'd like to chime in as a dyslexic myself and things I learned:

Monotonous walls of text are no good for anybody.

*Rich text* is fantastic. That is: Add legibility to paragraphs of copy by sprinkling different font weights (e.g. bold), italics and colour.

Underlines can be bad for legibility – if they do not skip ink (e.g. broken up from glyph descenders), otherwise if used sparingly as part of rich text they can be pretty great as well.

Thus practices, or dogma, that outlaw potential for variability in text are just a really bad idea.

The pseudo-italics can be bad if used in micro-copy or if applied monotonously across a whole paragraph.

**Comment by ericwbailey** [comment]
Closing this issue for inactivity, thank you to everyone for their contributions. Please feel free to re-open the issue if you would like to see the article published.
